README.md

```diff
@@ -135,7 +135,7 @@ and run one of the following commands:
 ## Future Work
 * Add ERC-20 Rosetta Module to enable reading ERC-20 token transfers and transaction construction
 * [Rosetta API `/mempool/*`](https://www.rosetta-api.org/docs/MempoolApi.html) implementation
-* Add more methods to the `/call` endpoint (currently only supports `eth_getBlockByNumber`,  `eth_call`, and `eth_getTransactionReceipt`)
+* Add more methods to the `/call` endpoint (currently only supports `eth_getBlockByNumber`,  `eth_call`, `eth_estimateGas`, and `eth_getTransactionReceipt`)
```

Comment:
Construction metadata endpoint of the implementation also give you suggested fee. Is this endpoint different?

Reply:
Yep, it makes a call to the contract to see how much fee is required for the particular token transfer.

Reply:
I see. so this is for token specific transfers. okay